server/sonar-server/src/test/java/org/sonar/server/test/index/TestIndexTest.java

```diff
@@ -0,0 +1,111 @@
+/*
+ * SonarQube, open source software quality management tool.
+ * Copyright (C) 2008-2014 SonarSource
+ * mailto:contact AT sonarsource DOT com
+ *
+ * SonarQube is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * SonarQube is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+
+package org.sonar.server.test.index;
+
+import org.junit.Before;
+import org.junit.ClassRule;
+import org.junit.Test;
+import org.sonar.api.config.Settings;
+import org.sonar.server.es.EsTester;
+
+import java.util.*;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class TestIndexTest {
+  @ClassRule
+  public static EsTester es = new EsTester().addDefinitions(new TestIndexDefinition(new Settings()));
+
+  TestIndex sut = new TestIndex(es.client());
+
+  @Before
+  public void before() throws Exception {
+    es.truncateIndices();
+  }
+
+  @Test
+  public void lines_covered_a_test_method() throws Exception {
+    List<Map<String, Object>> coverageBlocks = new ArrayList<>();
+    coverageBlocks.add(newCoverageBlock("1"));
+    coverageBlocks.add(newCoverageBlock("2"));
+    coverageBlocks.add(newCoverageBlock("3"));
```

Comment:
I In fact the these 4 lines are useless, please remove them (same for next tests)